src/app/store/technical-records/actions/technical-record-service.actions.ts

```diff
@@ -52,6 +52,10 @@ export const updateEditingTechRecord = createAction(`${prefix} updateEditingTech
 export const updateEditingTechRecordCancel = createAction(`${prefix} updateEditingTechRecordCancel`);
 
 export const changeVehicleType = createAction(`${prefix} changeVehicleType`, props<{ vehicleType: VehicleTypes }>());
+//not sure on this
+export const amendVin = createAction(`${prefix} amendVin`, props<{ oldVin: string; newVin: string; systemNumber: string }>());
+export const amendVinSuccess = createAction(`amendVin Success`);
```

Comment:
```suggestion
export const amendVinSuccess = createOutcomeAction(`amendVin`, true);
```

Reply:
I wasn't sure with this one, As the API just responds with a 200 and ok and not a tech record I'd kept the createAction as createOutcomeSuccess seems to respond with an array of vehicle tech records which I didn't think was correct for this action?